Packs/DeepInstinct/Integrations/DeepInstinct/DeepInstinct.yml

```diff
@@ -0,0 +1,179 @@
+commonfields:
+  id: Deep Instinct
+  version: -1
+name: Deep Instinct
+display: Deep Instinct
+category: Network Security
+description: Deep Instinct
+configuration:
+- display: base server url
+  name: base_url
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: api key
+  name: apikey
+  defaultvalue: ""
+  type: 4
+  required: true
+- display: Fetch incidents
+  name: isFetch
+  type: 8
+  required: false
+- display: Incident type
+  name: incidentType
+  type: 13
+  required: false
+- display: first event id to fetch from
+  name: first_fetch_id
+  defaultvalue: "0"
+  type: 0
+  required: false
+script:
+  commands:
+  - name: di-get-device
+    arguments:
+    - name: device_id
+      required: true
+      description: The device id
+    outputs:
+    - contextPath: DeepInstinct.id
```

Comment:
this is the wrong path should be 
`DeepInstinct.devices.id`, in the rest of the context paths as well
And I recommend using ID rather than id